llvm/include/llvm/CodeGen/GlobalISel/Utils.h

```diff
@@ -337,11 +337,12 @@ bool isKnownToBeAPowerOfTwo(Register Val, const MachineRegisterInfo &MRI,
 /// Returns true if \p Val can be assumed to never be a NaN. If \p SNaN is true,
 /// this returns if \p Val can be assumed to never be a signaling NaN.
 bool isKnownNeverNaN(Register Val, const MachineRegisterInfo &MRI,
-                     bool SNaN = false);
+                     GISelValueTracking *ValueTracking, bool SNaN = false);
```

Comment:
If this is optional it should be default initialized to null. If it's required it should be a reference. It should probably be required?